docs/rules/attribute-hyphenation.md

```diff
@@ -35,7 +35,8 @@ This rule enforces using hyphenated attribute names on custom components in Vue
 ```json
 {
   "vue/attribute-hyphenation": ["error", "always" | "never", {
-    "ignore": []
+    "ignore": [],
+    "includeSlots": false,
```

Comment:
I think the default should be `true`, to be honest, but I don't know the way that we release breaking changes. I thought it would be better to put the behavior behind an option, and then we can change the defaults as part of a major version, should we decide that's the best default.

Reply:
Thank you for this PR!

I think it's a bug that `<slot>` is ignored from the check, so I think it's a good to remove the option and always check `<slot>`.